buildenv/jenkins/variables/defaults.yml

```diff
@@ -300,10 +295,6 @@ x86-64_linux:
 #========================================#
 x86-64_linux_cm:
   extends: ['x86-64_linux', 'cmake']
-  excluded_tests:
```

Comment:
Pls keep these (and special.system) excluded for Java 8 until such time as we're ready to switch this version. We only have so many machines, and want to get through the nightly testing.